test/integration/test_integration.py

```diff
@@ -1593,15 +1593,28 @@ def test_autoquant_one_input(self, device, dtype, m, k, n):
     @unittest.skipIf(not TORCH_VERSION_AT_LEAST_2_5, "autoquant requires 2.5+.")
     def test_autoquant_compile(self, device, dtype, m1, m2, k, n):
         undo_recommended_configs()
-        if device != "cuda" or not torch.cuda.is_available():
+
+        # Check if we're running on a supported device
+        is_cuda_available = torch.cuda.is_available()
+        is_rocm_available = torch.version.hip is not None
+        is_supported_device = device == "cuda" and (
```

Comment:
wait, is rocm using the same "cuda" device?

Reply:
The "cuda" device here implies GPUs. Just pushed to make it more readable.